config/environments/production.rb

```diff
@@ -76,4 +76,28 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # mailer
+  config.action_mailer.default_url_options = { :host => 'immense-oasis-2395.heroku.com' }
+  # Don't care if the mailer can't send.
+  config.action_mailer.delivery_method = :sendmail
+  # Defaults to:
+  # config.action_mailer.sendmail_settings = {
+  #   location: '/usr/sbin/sendmail',
+  #   arguments: '-i -t'
+  # }
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :address              => "smtp.live.com",
+    :port                 => 587,
+    :enable_starttls_auto => true,
+    :user_name            => 'countries_launch_academy@zoho.com',
+    :password             => 'MakeNoBugs!',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.